### PR TITLE
Added scripts for running windbreaker services together locally

### PR DESCRIPTION
**Pull request by ctdio**
I added some scripts and a compose file to help launch all of the services we create locally. This can be used to ensure that services are actually communicating with each other properly. Eventually it would be nice to get end to end tests up and running.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - postgres
+      - rabbitmq
+  postgres:
+    image: 'postgres:9.6.3-alpine'
+    ports:
+      - 5432
+  rabbitmq:
+    image: 'rabbitmq:3.6.10-alpine'
+    ports:
+      - 5672
```
Comment (ctdio) [inline docker-compose.yml:58]: I originally wanted to use docker's ability to layer compose files, but right now there's a limitation on how build contexts are resolved that are causing issues with builds.

Comment (Druotic) [reply docker-compose.yml:58]: Is this a known limitation that has an associated github issue, or just a general limitation that isn't expected to change?

File: deploy-local.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker-compose up windbreaker rss-watcher
```
Comment (Druotic) [inline deploy-local.sh]: Are the args `windbreaker rss-watcher` needed here?

File: initialize-repos.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+}
+
+mkdir repos
+
+set -e
+
+# clone projects into repos folder
+clone windbreaker
+clone rss-watcher
+clone npm-watcher
```
Comment (Druotic) [inline initialize-repos.sh]: This works for now, but in the future we'll probably want to pull updates if the repos have already been cloned previously.

File: deploy-local.sh (hunk 2 above)
Comment (ctdio) [reply deploy-local.sh]: No, but just wanted to be explicit in case we later add in a `test` container

File: initialize-repos.sh (hunk 3 above)
Comment (ctdio) [reply initialize-repos.sh]: I agree

Comment (austinkelleher) [reply initialize-repos.sh]: I think this really is as simple as:

```
clone () {
  if cd repo; then git pull && cd ../; else git clone git@github.com:windbreaker-io/$1 repos/$1; fi
}
```

**Review by austinkelleher — CHANGES_REQUESTED** [review]

File: initialize-repos.sh (hunk 3 above)
Comment (Epitrochoid) [reply initialize-repos.sh]: +1 on the bash function, though `if [[ -d "repo/.git" ]]...` is how I would do it, I didn't actally know that you could `if` on cd.

**Review by Epitrochoid — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (ctdio) [reply docker-compose.yml:58]: not a problem anymore

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+      #  - ../windbreaker-service-util:/usr/npm-watcher/node_modules/windbreaker-service-util
+    environment:
+      SERVICE_ENVIRONMENT: docker
+    command: npm start
+
+
+  # a "grouping" container that is here to keep the
+  # compose file a little more dry
+  dependencies:
+    image: tianon/true
```
Comment (ctdio) [inline docker-compose.yml:46]: Super tiny container just to group dependencies together. As we add more services and this docker file will get filled with services containing a lot of the same docker service configuration. It's only 125 bytes and exits out immediately, so I figured it's not a big deal.

**Comment by ctdio** [comment]
@Druotic @austinkelleher pls rereview

**Comment by ctdio** [comment]
Btw, if you are doing a lot of cross service work, I recommend just working from within this repo. You can make changes in each of the individual services that get pulled down and validate that communication is working correctly from the root of this repo. Later on we can add a `test` container that runs integration tests.

**Review by Druotic — APPROVED** [review]

**Pull request merged**